local-cli/runIOS/runIOS.js

```diff
@@ -52,10 +52,10 @@ function _runIOS(argv, config, resolve, reject) {
     throw new Error(`Cound't find ${args.simulator} simulator`);
   }
 
-  const simulatorFullName = `${selectedSimulator.name} (${selectedSimulator.version})`;
+  const simulatorFullName = `${selectedSimulator.name} (${selectedSimulator.version}) [${selectedSimulator.udid}]`;
```

Comment:
That's true, but on the other hand - since we launch simulators based on the string (name), not its udid, I am not sure if it helps that much on regular basis. If you have say naming conflict, I am pretty sure the first device from the list will get selected.